File: spatial-id-py/example_generate_id.py
from spatial_id_py import generate_spatial_id

if __name__ == "__main__":
    lat, lon, alt, zoom = 0.0, 0.0, 10.0, 25
    spatial_id = generate_spatial_id(lat, lon, alt, zoom)
    print(f"Spatial ID: {spatial_id}")

File: examples/typescript/example_generate_id.ts
import { generate_spatial_id } from 'spatial-id-js';

const output = document.getElementById('output');
try {
  const lat = 0.0;
  const lon = 0.0;
  const alt = 10.0;
  const zoom = 25;
  const spatialId = generate_spatial_id(lat, lon, alt, zoom);
  if (output) {
    output.textContent = `Spatial ID: ${spatialId}`;
  }
  console.log('Spatial ID:', spatialId);
} catch (e) {
  if (output) output.textContent = `エラー: ${String(e)}`;
  console.error(e);
}
